example/main.go

```diff
@@ -43,42 +43,41 @@ func main() {
 	resolveTOTPSecretReference(client, item.VaultID, item.ID, "TOTP_onetimepassword")
 	sharelink := generateItemSharing(client, item.VaultID, item.ID)
 	fmt.Println(sharelink)
+	archiveItem(client, item.VaultID, item.ID)
 	deleteItem(client, item.VaultID, item.ID)
 }
 
 func listVaultsAndItems(client *onepassword.Client, vaultID string) {
 	// [developer-docs.sdk.go.list-vaults]-start
-	vaults, err := client.Vaults().ListAll(context.Background())
+	vaults, err := client.Vaults().List(context.Background())
 	if err != nil {
 		panic(err)
 	}
-	for {
-		vault, err := vaults.Next()
-		if errors.Is(err, onepassword.ErrorIteratorDone) {
-			break
-		} else if err != nil {
-			panic(err)
-		}
-
+	for _, vault := range vaults {
```

Comment:
`[non-blocking remark]`

This feels much more natural. Thanks for the improvement.